openssl/src/ssl/tests.rs

```diff
@@ -344,19 +465,20 @@ run_test!(get_peer_certificate, |method, stream| {
 #[test]
 #[cfg(feature = "dtlsv1")]
 fn test_write_dtlsv1() {
-    let sock = UdpSocket::bind("127.0.0.1:0").unwrap();
-    let stream = sock.connect("127.0.0.1:15410").unwrap();
+    let yes = iter::repeat("y\n").take(1_000_000).collect::<String>();
```

Comment:
This is probably the sketchiest part of the tests, it previously was a pipe of `yes` into an `openssl s_server`, but now it's just a whole lot of `y` lines. Looks like it's passing the bots now though but I could switch to a true infinite loop if needed.